### PR TITLE
Commit hook (second batch)

### PR DESCRIPTION
**Pull request by martin-ueding**
I just realized that `git diff` does not tell us whether there are any new files. So I now also make sure that the output of `git status -s` has not changed.

Additionally only symlinks that do not exist yet are created.

### CODE REVIEW

**Comment by kostrzewa** [comment]
Shouldn't the status call be ```-us``` ?

**Comment by kostrzewa** [comment]
I mean ```-u```

**Comment by kostrzewa** [comment]
I have

```
git config --global status.showUntrackedFiles 
no
```

set, so ```git status -s``` does not show untracked files.

**Comment by martin-ueding** [comment]
It seems that the default value for `-u` is `all`. But yes, with the change in the configuration we need to give that.

**Comment by j-ostmeyer** [comment]
But -u shows a lot of things that we are not interested in.

File: hooks/pre-commit [hunk 1]
```diff
@@ -13,19 +13,30 @@ reset="$(tput sgr0)"
...
   rm -f "$diff_old"
   rm -f "$diff_new"
+  rm -f "$status_old"
+  rm -f "$status_new"
 }
 
 trap cleanup EXIT
 
 git diff > "$diff_old"
+git status --porcelain > "$status_old"
```
Comment (kostrzewa) [inline hooks/pre-commit]: nope, doesn't work for me

Comment (kostrzewa) [reply hooks/pre-commit]: only if I add ```-u``` in addition does this work

**Comment by martin-ueding** [comment]
Huh, that seems to be against the manual:

```
--porcelain[=<version>]
           Give the output in an easy-to-parse format for scripts. This is similar to the short output, but will
           remain stable across Git versions and regardless of user configuration. See below for details.

           The version parameter is used to specify the format version. This is optional and defaults to the
           original version v1 format.
```

Okay, then we also specify `-u`.

**Pull request merged**